grammar.ebnf

```diff
@@ -20,8 +20,19 @@ StructDecl = "struct", Identifier, "{", { StructField }, "}";
 StructField = Identifier, ":", Type, ";";
 
 
+Statement = "{", { Statement }, "}"
+          | Type, Identifier, ";"
+          | "while", "(", Expression, ")", "{", { Statement }, "}"
+          | Lval, "=", Expression, ";"
+          | Lval, "[", Expression, "]", "=", Expression, ";"
+          | "if", "(", Expression, ")", Statement;
+
+
+Lval = Identifier, { ".", Identifier };
```

Comment:
This is a good thought but I propose that we restrict that to just `Identifier` in the start. Then we can start doing more complicated things.

Reply:
Fair enough.